task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -155,19 +155,19 @@ spec:
         ssh -v $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/workspaces" "$BUILD_DIR/scripts" "$BUILD_DIR/tmp" "$BUILD_DIR/tekton-results" "$BUILD_DIR/entitlement"
 
         if [ ! -n "${CONFIG_TOML_FILE}" ]; then
-          echo "No CONFIG_TOML_FILE specified"
+          echo "No CONFIG_TOML_FILE specified. Assuming config.toml"
           export CONFIG_TOML_FILE=config.toml
-          if [ -f /var/workdir/source/config.toml ]; then
-            echo "Using the config.toml file found in the repository root!"
-            echo "  Remove the config.toml file or set params.CONFIG_TOML_FILE to another file to prevent using config.toml."
-          else
-            echo "No config.toml file found. Using an empty configuration."
-            touch /var/workdir/source/$CONFIG_TOML_FILE
-          fi
         fi
-        echo "Using the following config.toml file $CONFIG_TOML_FILE:"
-        cat /var/workdir/source/$CONFIG_TOML_FILE
 
+        if [ -f "/var/workdir/source/${CONFIG_TOML_FILE}" ]; then
+          echo "Using the ${CONFIG_TOML_FILE} file found in the repository root!"
+        else
+          echo "No ${CONFIG_TOML_FILE} file found in the repository. Set params.${CONFIG_TOML_FILE}_FILE to a valid config.toml file."
+          exit 1
```

Comment:
This will make it impossible for the task to run if there is no file regardless as to whether the parameter was specified or not. Is that desired?

Reply:
I think so. Right? Like, what's the behavior of bib if there is literally no config.toml at all?

Reply:
This is effectively a breaking change. If we want to make it explicit like this then will we have to bump the version?

Reply:
I don't know if the file is missing, but bib is fine if the file is empty

Reply:
OK - I revised this to drop the `exit 1` and restore the `touch` line. (Now I understand what that line was for. I assumed bib would balk at an empty config.toml file.)